src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -39,17 +39,40 @@ const numberToWords = {
   9: 'Tenth',
 };
 
-// This removes "First " from the title if there is only one incident.
+const REVIEW_TITLE_TOKEN = '[index]';
+
+/**
+ * This removes "First " from the title if there is only one incident.
+ *
+ * @param {string} [title] Displayed as the section summary header.
+ * If contains REVIEW_TITLE_TOKEN and there is more than one incident, replaces REVIEW_TITLE_TOKEN with numberToWords[index].toLowerCase().
+ * If does not contain REVIEW_TITLE_TOKEN appends numberToWords[index] to front of title.
+ * @param {int} index Index of numberToWords
+ * @param {string} formType Indicates what type of form is calling function; 781, 781a
+ * @returns {object} title
+ */
 const setReviewTitle = (title, index, formType) => formData => {
+  const additionalIncidentKeyIndex = index === 0 ? index : index - 1;
   const additionalIncidentKey = `view:enterAdditional${
     formType === '781a' ? 'Secondary' : ''
-  }Events${index}`;
+  }Events${additionalIncidentKeyIndex}`;
 
   let formattedTitle = title;
 
   if (formData[additionalIncidentKey]) {
-    formattedTitle = `${numberToWords[index]} ${title}`;
+    if (title.search(REVIEW_TITLE_TOKEN) > 0) {
+      formattedTitle = title.replace(
+        REVIEW_TITLE_TOKEN,
+        ` ${numberToWords[index].toLowerCase()} `,
+      );
+    } else {
+      // If does not contain REVIEW_TITLE_TOKEN put numberToWords[index] at start of title
+      formattedTitle = `${numberToWords[index]} ${title}`;
```

Comment:
Same ? here.

Reply:
can't have more than 3 additional events during the 781/a flow